constants/src/main/java/org/jboss/pnc/enums/SystemImageType.java

```diff
@@ -24,13 +24,19 @@
  */
 public enum SystemImageType {
 
-    /** A Docker image will be used to run the build */
+    /**
+     * A Docker image will be used to run the build.
```

Comment:
A Docker-formatted image will be used to create a container where to run the build.